gnome-base/gnome-shell/gnome-shell-45_beta1.ebuild

```diff
@@ -0,0 +1,183 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit gnome.org gnome2-utils meson optfeature python-single-r1 virtualx xdg
+
+DESCRIPTION="Provides core UI functions for the GNOME desktop"
+HOMEPAGE="https://wiki.gnome.org/Projects/GnomeShell https://gitlab.gnome.org/GNOME/gnome-shell"
+
+LICENSE="GPL-2+ LGPL-2+"
+SLOT="0"
+IUSE="cameramonitor elogind gtk-doc +ibus +networkmanager systemd test"
```

Comment:
You're adding `IUSE=cameramonitor` but making the dependency on pipewire unconditional.

I'd suggest just naming the USE flag "pipewire" and making the pipewire dep conditional on that.